### PR TITLE
 Prepare for release 3.0.0

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

- Updated version to 3.0.0
- Executed antsibull-changelog to generate changelog

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by alinabuzachis — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -762,3 +762,53 @@ releases:
...
+  3.0.0:
+    changes:
+      breaking_changes:
+      - Remove support for ansible-core < 2.14
+      - Update python kubernetes library to 24.2.0 , helm/kind-action to 1.8.0, kubernetes
+        >= 1.24.
+      bugfixes:
+      - helm - Put the chart_ref into quotes when running ``helm show chart``, ``helm
+        upgrade`` and ``helm dependency update`` commands (https://github.com/ansible-collections/kubernetes.core/issues/653).
+      - helm - delete temporary file created when deploying chart with option release_values
```
Comment (hakbailey) [inline changelogs/changelog.yaml:774]: For the other releases we've been going through the new changelog entries and making sure that all options are monospace formatted like below, do we want to do that for this release too?

```suggestion
      - helm - delete temporary file created when deploying chart with option ``release_values``
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
...
+
+Minor Changes
+-------------
+
+- helm - add reuse_values and reset_values support to helm module (https://github.com/ansible-collections/kubernetes.core/issues/394).
+- k8s - add new option delete_all to support deletion of all resources when state is set to absent. (https://github.com/ansible-collections/kubernetes.core/issues/504)
+- k8s, k8s_info - add a hidden_fields option to allow fields to be hidden in the results of k8s and k8s_info
+- k8s_drain - add ability to filter the list of pods to be drained by a pod label selector (https://github.com/ansible-collections/kubernetes.core/issues/474).
+
+Breaking Changes / Porting Guide
```
Comment (hakbailey) [inline CHANGELOG.rst:25]: Should we move the Breaking Changes section to the top (in config.yaml)?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
 .. contents:: Topics
 
 
+v3.0.0
+======
+
+Release Summary
+---------------
+
+This major release drops support for ansible-core versions lower than 2.14, Python versions lower than 3.9 and updates python kubernetes library to 24.2.0 , helm/kind-action to 1.8.0, kubernetes >= 1.24, along with bug fixes and minor changes.
```
Comment (alinabuzachis) [inline CHANGELOG.rst:14]:
```suggestion
This major release drops support for ansible-core versions lower than 2.14, Python versions lower than 3.9 and updates python kubernetes library to 24.2.0, helm/kind-action to 1.8.0, kubernetes >= 1.24, along with bug fixes and minor changes.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
...
+
+
+Minor Changes
+-------------
+
+- helm - add reuse_values and reset_values support to helm module (https://github.com/ansible-collections/kubernetes.core/issues/394).
+- k8s - add new option delete_all to support deletion of all resources when state is set to absent. (https://github.com/ansible-collections/kubernetes.core/issues/504)
+- k8s, k8s_info - add a hidden_fields option to allow fields to be hidden in the results of k8s and k8s_info
```
Comment (alinabuzachis) [inline CHANGELOG.rst:22]: Can you please link the PR or the issue here?
```suggestion
- k8s, k8s_info - add a ``hidden_fields`` option to allow fields to be hidden in the results of k8s and k8s_info.
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
...
+- k8s_drain - add ability to filter the list of pods to be drained by a pod label selector (https://github.com/ansible-collections/kubernetes.core/issues/474).
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- Remove support for ansible-core < 2.14
+- Update python kubernetes library to 24.2.0 , helm/kind-action to 1.8.0, kubernetes >= 1.24.
```
Comment (alinabuzachis) [inline CHANGELOG.rst:29]:
```suggestion
- Update python kubernetes library to 24.2.0, helm/kind-action to 1.8.0, kubernetes >= 1.24.
```

File: CHANGELOG.rst (hunk 2 above)
Comment (GomathiselviS) [reply CHANGELOG.rst:25]: Is this ordering done by antsibull-changelog tool? I see this is the order maintained in the previous releases as well.

Comment (hakbailey) [reply CHANGELOG.rst:25]: Yeah it is but the order is set by the order of sections in the changelog/config.yaml file.

Comment (hakbailey) [reply CHANGELOG.rst:25]: I don't know why I thought I saw that at the top in other collections but now I'm not seeing so I don't know where that came from :)

File: CHANGELOG.rst [hunk 6]
```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
 .. contents:: Topics
 
 
+v3.0.0
```
Comment (hakbailey) [inline CHANGELOG.rst:8]: I think this needs to be regenerated to get the updates from the changelog.yaml file?

**Review by hakbailey — APPROVED** [review]

**Pull request merged**